### ISSUE TITLE
Provide way to bring back ✔️ in chat response list

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#297114]
Users with low vision or intellectual disablities will struggle to take in the shimmer effect that indicates chat progress. 

We should have a setting that enables users to bring ✔️s back.
